polling_stations/apps/data_collection/management/commands/import_swale.py

```diff
@@ -1,53 +1,13 @@
-from django.contrib.gis.geos import Point
 from data_collection.management.commands import BaseHalaroseCsvImporter
 
 
 class Command(BaseHalaroseCsvImporter):
     council_id = "E07000113"
     addresses_name = (
-        "europarl.2019-05-23/Version 1/polling_station_export-2019-05-06Swale.csv"
+        "parl.2019-12-12/Version 1/polling_station_export-2019-11-04swale.csv"
     )
     stations_name = (
-        "europarl.2019-05-23/Version 1/polling_station_export-2019-05-06Swale.csv"
+        "parl.2019-12-12/Version 1/polling_station_export-2019-11-04swale.csv"
     )
-    elections = ["europarl.2019-05-23"]
-
-    def address_record_to_dict(self, record):
-        rec = super().address_record_to_dict(record)
-        uprn = record.uprn.strip().lstrip("0")
-
-        if uprn in [
-            "200001875126",  # ME122HP -> ME122LZ : Morland House Augustine Road, Minster-on-Sea, Sheerness, Kent
-            "100061078702",  # ME122LE -> ME122LA : Uppal Villa Minster Drive, Minster-on-Sea, Sheerness, Kent
-            "100061081138",  # ME122LS -> ME122LX : 150 Scarborough Drive, Minster-on-Sea, Sheerness, Kent
-            "10023197317",  # ME122SG -> ME122SH : 1A Baldwin Cottage Baldwin Road, Minster-on-Sea, Sheerness, Kent
-            "10023197934",  # ME122LT -> ME122LX : 152 Scarborough Drive, Minster-on-Sea, Sheerness, Kent
-            "100061080835",  # ME123JE -> ME123HT : 1A Rosemary Avenue, Minster-on-Sea, Sheerness, Kent
-            "10023200030",  # ME103TU -> ME123TU : 28 Petunia Avenue, Minster-on-Sea, Sheerness, Kent
-            "10035061220",  # ME121AG -> ME122AG : 178 Invicta Road, Sheerness, Kent
-            "10093083738",  # ME124JB -> ME101QA : Flat Above Marinos Fish Bar 212 London Road, Sittingbourne, Kent
-            "100061078990",  # ME123PA -> ME123NZ : 382B Minster Road, Minster-on-Sea, Sheerness, Kent
-            "100061083074",  # ME122SG -> ME122SD : Llamedos The Glen, Minster-on-Sea, Sheerness, Kent
-            "100062379223",  # ME124JA -> ME124JB : Sheringham Bell Farm Lane, Minster-on-Sea, Sheerness, Kent
-            "100061073637",  # ME124JA -> ME124JB : The Laurels Bell Farm Lane, Minster-on-Sea, Sheerness, Kent
-            "100061073623",  # ME124JA -> ME124JB : Merry Moments Bell Farm Lane, Minster-on-Sea, Sheerness, Kent
-            "200002539987",  # ME101NL -> ME101NS : Flat L 94 London Road, Sittingbourne, Kent
-        ]:
-            rec["accept_suggestion"] = True
-
-        if uprn in [
-            "100062087806",  # ME121TP -> ME122RT : Flat 2 36A Broadway, Sheerness, Kent
-            "100062087803",  # ME121TP -> ME122RT : Flat 1 36A Broadway, Sheerness, Kent
-            "10023196555",  # ME122DH -> ME121AG : 3 The Crescent Parklands Village The Broadway, Minster-on-Sea, Sheerness, Kent
```

Comment:
Similar comment to Hambleton. Although the list of errors this time is different, there are definitely some of these that are valid to carry forward. If we've already made the decisions about them, it would be useful to hold on to the ones that are still relevant.